setup.py

```diff
@@ -1147,7 +1147,7 @@ def main():
         "filelock",
         "typing-extensions>=4.8.0",
         'setuptools ; python_version >= "3.12"',
-        'sympy==1.13.1 ; python_version >= "3.9"',
+        'sympy>=1.13.1,!=1.13.2 ; python_version >= "3.9"',
```

Comment:
Not sure what other versions between 1.13.1 and 1.32.3 you want to install if 1.13.2 is broken
```suggestion
        'sympy>=1.13.3 ; python_version >= "3.9"',
```

Reply:
Expected versions 1.13.1 (included) and 1.13.3+ can be supported, but I'm willing to forgo support for 1.13.1 if needed.